MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include zamba/models/official_models/*/config.yaml
+include zamba/models/official_models/*/hparams.yaml
```

Comment:
Any reason not to just distribute the whole set of files by default? Are we confident these are the only ones we'll ever reference?

Reply:
Options are:
1. include just the yaml files we need
2. include all the yaml files (includes the train_configuration and predict_configuration that are just references of the full set of parameters we used in training the official models)
3. include all the files in the official models directory (also includes the val_metrics.json)

The difference between 2 and 3 doesn't seem important. I'm not sure how to think about the difference between 1 and 2. The downside of 1 is that it's easy to miss including files we should include if we add more yaml files in the future that are referenced. The upside is that we won't end up with obtuse errors where zamba tries to load a config that has machine specific information in it that was never intended for public use. What do you think?

Reply:
Good points. I guess I've seen more errors from thinking we were doing 1 right then 2.

Reply:
Sounds good, that will match the behavior we do for object detection and densepose.